ios/brave-ios/Sources/Brave/Frontend/Browser/Tab.swift

```diff
@@ -67,6 +67,17 @@ protocol URLChangeDelegate {
   func tab(_ tab: Tab, urlDidChangeTo url: URL)
 }
 
+struct RewardsTabChangeReportingState {
+  /// Set to true when the resulting page was restored from session state.
+  var isSessionStateRestored = false
```

Comment:
nit: now that this variable is a part of `RewardsTabChangeReportingState` we could just name this variable `wasRestored`, your thoughts?

Reply:
Fixed